YoCode/ProjectRunner.cs

```diff
@@ -9,16 +11,39 @@ public class ProjectRunner
 
         // TODO: find other way of running .dll file instead of hardcoding the name 
         // TODO: find a way to specify location of appsettings.json file when running
-        private string Argument { get; } = @"bin\Debug\netcoreapp1.1\UnitConverterWebApp.dll";
+        private string Argument { get; } = @"bin\Debug\";   //      netcoreapp2.0\UnitConverterWebApp.dll";
         public string Output { get; }
         private string ErrorOutput { get; }
 
         public ProjectRunner(string workingDir)
         {
+            workingDir += @"\UnitConverterWebApp";
+
+            Argument = Argument + (Path.GetFileName(Directory.GetDirectories(workingDir + "\\" + Argument).First()))+"\\UnitConverterWebApp.dll";
```

Comment:
This seems like a good solution. Choose the first one because there's likely to only be one anyway. It's very unlikely a single project would be built with 2 different versions of .Net Core. Good thinking. 👍

Reply:
You could use `Path.Combine` instead of manually joining strings with `+` but it doesn't really matter if we're always going to run on Windows.